airtime_mvc/application/common/FileIO.php

```diff
@@ -70,11 +70,17 @@ public static function smartReadFile($filePath, $size, $mimeType)
             ob_end_flush();
         }
 
-        // NOTE: We can't use fseek here because it does not work with streams
-        // (a.k.a. Files stored in the cloud)
-        while(!feof($fm) && (connection_status() == 0)) {
+
+        //These two lines were removed from Airtime 2.5.x at some point after Libretime forked from Airtime.
```

Comment:
We went with the last SaaS code drop from sourcefabric when forking since the SaaS version had way more work put into it than 2.5.x.